cassandra/src/test/java/org/apache/calcite/test/CassandraExtension.java

```diff
@@ -175,14 +189,6 @@ private CassandraResource() {
         Thread.currentThread().interrupt();
       }
       Stage.shutdownNow();
-
-      if (FBUtilities.isWindows) {
```

Comment:
Is this because Cassandra dropped Windows?

Reply:
That's correct, those were Windows-specific methods that C* dropped when they stopped supporting it